source/TargetEntity.h

```diff
@@ -156,6 +156,10 @@ class TargetEntity : public VisibleEntity {
 		return pow(1.0f + TARGET_MODEL_ARRAY_SCALING, m_scaleIdx - TARGET_MODEL_ARRAY_OFFSET);
 	}
 
+	String id() {
```

Comment:
should this be `const String &id() {`?

Reply:
Sure, moreover it should _really_ be:  `const String& id() const {`

Reply:
[This commit](https://github.com/NVlabs/abstract-fps/pull/105/commits/6c3190cc1283e8cdaf0dee19ce9dc7b473604a6d) improves the use of `const` around `TargetEntity` member methods.